mysql/util.go

```diff
@@ -93,3 +97,11 @@ func GetDefaultFieldLengthAndDecimalForCast(tp byte) (flen int, decimal int) {
 	}
 	return -1, -1
 }
+
+// GetDefaultExplainFormat return the processed values for format of explain
+func GetDefaultExplainFormat(f string) string {
```

Comment:
Isn't `row` is returned for `TRADITIONAL`?

https://github.com/pingcap/parser/blob/f88b9d7642986fb6f5c4f327bb346b6b2fb1aa63/parser.y#L4463-L4466

I don't get it. Maybe we could replace `stringLit` into `ExplainFormatType`? May you investigate why it is `stringLit` rather than `ExplainFormatType`?

Reply:
In fact, the type of parts is different. the first is Identifier. The socend is a string. so we can support diffrent types in expalain_format. In goyacc, I don't find a function that it can get value from different type. but in MySQL, ``` ident_or_text``` can do it.

Reply:
it is possible for it to improve, But I think it is a appropriate way to create a new PR for it to improve.

Reply:
What's the purpose exporting this function, is it used anywhere else?